src/components/tooltip/Tooltip.tsx

```diff
@@ -19,205 +17,187 @@ const setColor = (color: string) => {
     : color;
```

Comment:
The setColor is common in application let's put it into utils, and import where needed.